### ISSUE TITLE
Home Assistant 2021.3.0 breaks Magic Areas

### ISSUE THREAD

**Issue by TheArcturian** [jseidl/hass-magic_areas#103]
Just installed the latest 2021.3.0 version of HA. This breaks Magic Areas with the following error in the logs:

```
Logger: homeassistant.setup
Source: custom_components/magic_areas/__init__.py:63
First occurred: 12:12:17 AM (1 occurrences)
Last logged: 12:12:17 AM

Error during setup of component magic_areas
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 213, in _async_setup_component
    result = await task
  File "/config/custom_components/magic_areas/__init__.py", line 63, in async_setup
    areas.append(AreaEntry(name=meta_area, id=meta_area.lower()))
TypeError: __init__() missing 1 required positional argument: 'normalized_name'

```

**Comment by klatka** [comment]
Confirm

**Comment by danielbrunt57** [comment]
Ditto.
```
Logger: homeassistant.setup
Source: custom_components/magic_areas/__init__.py:63
First occurred: 7:09:02 PM (1 occurrences)
Last logged: 7:09:02 PM

Error during setup of component magic_areas
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 213, in _async_setup_component
    result = await task
  File "/config/custom_components/magic_areas/__init__.py", line 63, in async_setup
    areas.append(AreaEntry(name=meta_area, id=meta_area.lower()))
TypeError: __init__() missing 1 required positional argument: 'normalized_name'
```

**Comment by danielbrunt57** [comment]
I hacked this into __init__.py line #63:
```
        areas.append(AreaEntry(name=meta_area, normalized_name=meta_area, id=meta_area.lower()))^M
```
Not sure if it's correct or not but MA is working again!

**Comment by mwolter805** [comment]
@danielbrunt57 Thanks for sharing that. Removed the ^M from the end and MA appears to be working.

**Comment by jseidl** [comment]
Released fix on 2.0.4. Thank you! https://github.com/jseidl/hass-magic_areas/releases/tag/v2.0.4
